pkg/config/config.go

```diff
@@ -24,11 +24,19 @@ import (
 	"github.com/ghodss/yaml"
 	"io/ioutil"
 	"os"
+	"time"
 )
 
-const V3ioConfigEnvironmentVariable = "V3IO_CONF"
-const DefaultConfigurationFileName = "v3io.yaml"
-const SCHEMA_CONFIG = ".schema"
+const (
+	V3ioConfigEnvironmentVariable = "V3IO_CONF"
+	DefaultConfigurationFileName  = "v3io.yaml"
+	SCHEMA_CONFIG                 = ".schema"
+
+	defaultNumberOfWorkers      = 1
+	defaultNumberOfQueryWorkers = 8
+	defaultBatchSize            = 64
+	defaultTimeout              = 24 * time.Hour
```

Comment:
Now the units are in nanosecond instead of seconds. should change documentation in `v3io.yaml.template` and in `config.go`. Or change it here to seconds.